### ISSUE TITLE
Question about InfoNCE loss

### ISSUE THREAD

**Issue by hotchilipowder** [wujcan/SGL-Torch#3]
Hello, I am working on this project. I'm not quite sure about the InfoNCE loss when I was looking at this code. The torch.logsumexp function is used [here](https://github.com/wujcan/SGL-Torch/blob/c56328817daa6ebd7669cb9c0429a1f0f3dad5b9/model/general_recommender/SGL.py#L283), but it doesn't seem to have a negative sign added to it? In another tensorflow repository, however, there is a negative sign in [Line 290](https://github.com/wujcan/SGL-TensorFlow/blob/58fe93149ca6fb733639f556343c41f12206ba55/model/general_recommender/SGL.py#L290). 

Thank you

**Comment by wujcan** [comment]
Please see the return variables of the [forward](https://github.com/wujcan/SGL-Torch/blob/c56328817daa6ebd7669cb9c0429a1f0f3dad5b9/model/general_recommender/SGL.py#L61) function in class _LightGCN. Specifically, we have added the negative sign in [Line 91-92](https://github.com/wujcan/SGL-Torch/blob/c56328817daa6ebd7669cb9c0429a1f0f3dad5b9/model/general_recommender/SGL.py#L91).

**Comment by hotchilipowder** [comment]
Thank you for your reply
